osquery/filesystem/windows/fileops.cpp

```diff
@@ -98,6 +98,43 @@ class WindowsFindFiles {
   fs::path path_;
 };
 
+Status windowsShortPathToLongPath(const std::string& shortPath,
+                                  std::string& rLongPath) {
+  TCHAR longPath[MAX_PATH];
```

Comment:
I'm not sure I follow? A std::string can't take ownership of a buffer (afaict), and I need a writable char buffer for GetLongPathName to populate. If I use a std::array I'm still going to have to pass a raw pointer to it to the Windows API, and I'm still going to have to construct a new std::string to return.